repo_structure/repolint.json

```diff
@@ -6,8 +6,8 @@
     },
     "rules": {
       "all": {
-        "license-file-matches:file-hash": ["error", {"file": "LICENSE*", "hash":"b40930bbcf80744c86c46a12bc9da056641d722716c378f5659b9e555ef833e1"}],
-        "code-of-conduct-file-matches:file-hash": ["error", {"file": "CODE_OF_CONDUCT.md", "hash":"d2719e3ce1c383567a5f1bfe8f5c5af2042b3e15b5620b8d8a4b5ada022289e4"}],
+        "apache-license-file:file-contents": ["error", {"files": ["LICENSE*"], "content": "Apache License.*Version 2.0", "fail-on-non-existent": true}],
```

Comment:
The "magic words" approach gets to the heart of what we are looking for for an automated test.

Reply:
Agreed.